src/ExpenseTracker.Common/Repositories/Interface/IGenericRepository.cs

```diff
@@ -9,12 +9,13 @@ namespace ExpenseTracker.Common.Repositories.Interface
 {
     public interface IGenericRepository<T>
     {
+        Task<T> FindAsync(long id);
```

Comment:
@BirajMainali I think we can improve the `GenericRepo` API a bit.

1. Replace `GetById` with `Find`
2. Replace `GetByIdAsync` with  `FindAsync`